### ISSUE TITLE
Shortcuts throw errors

### ISSUE THREAD

**Issue by quadrophobiac** [theodi/comma-chameleon#144]
```
A JavaScript error occurred in the main process
Uncaught Exception:
ReferenceError: mainWindow is not defined
    at click 
```

**Comment by quadrophobiac** [comment]
errors occur via hotkeys or via manual use of electron edit menu

**Comment by quadrophobiac** [comment]
occurs in package or via npm start

**Comment by chris48s** [comment]
I think this issue describes the problem that PR #160 resolves

**Comment by chris48s** [comment]
Closed by #160 further discussion in #164
